MVVM/ViewModel/MainViewModel.cs

```diff
@@ -27,7 +27,7 @@ public partial class MainViewModel : Control
 		private ImageTexture? _imTexture;
 
 		[Export]
-		private TextureRect imTextureRect = null!;
+		private TextureRect _imTextureRect = null!;
```

Comment:
Rebase resulted in a conflict where some of the imTextureRect references were changed to the proper _... an the other were not. Changed all of them to _imTextureRect.

Reply:
My bad. Should be _...